src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -165,6 +165,8 @@ out.create=\
         \ \ created: {0}
 out.extracted=\
         extracted: {0}
+out.kept=\
+        \ \ skipped: {0}
```

Comment:
We might want to add a bit more wording to indicate it is being skipped due to the file already existing on disk

Reply:
I follow existing pattern with short status update. Open to suggestions.

Reply:
Perhaps something like:

` \ \  file: {0}  exists, skipped`

Reply:
Use this for now to be more close to current style,
`        \ \ skipped: {0} exists`
Was thinking perhaps a different line for detail message, but this should do, as we only skip with file exists.